library/cartridge_backup_instance.py

```diff
@@ -97,13 +101,31 @@ def backup_stop(control_console):
     return None
 
 
-def call_backup(params):
-    start_only = params['start_only']
-    stop_only = params['stop_only']
+def custom_backup(params):
+    custom_backup_files = params.get('custom_backup_files')
+    assert custom_backup_files is not None, 'Parameter "custom_backup_files" is required'
+
+    # ARCHIVE
+    instance_id = params.get('instance_id')
+    assert instance_id is not None, 'Parameter "instance_id" is required'
+    backups_dir = params.get('backups_dir')
+    assert backups_dir is not None, 'Parameter "backups_dir" is required'
+    backup_archive_path = backup_pack(instance_id, backups_dir, custom_backup_files)
+
+    return helpers.ModuleRes(changed=True, fact={
+        'backup_archive_path': backup_archive_path,
+        'backup_files': custom_backup_files,
+    })
+
+
+def tnt_backup(params):
```

Comment:
All backups here are tnt. Maybe it's "backup_running_instance" and "backup_stopped_instance"?

Reply:
One type of backups is provided by Tarantool box.backup(), another type of backups is provided by our custom list of files. So, when you are using box.backup(), you should start instance. When you are using out custom list of files, you should stop instance